README.md

```diff
@@ -50,7 +50,7 @@ Run the following command to install development image of OpenEBS velero-plugin
 This command will add an init container to Velero deployment to install the OpenEBS velero-plugin.
 
 ## Configuring snapshot location
-To take a backup of CStor volume through Velero, configure `VolumeSnapshotLocation` with provider `openebs.io/cstor-blockstore`. Sample YAML file for volumesnapshotlocation can be found at `example/06-volumesnapshotlocation.yaml`.
+To take a cloud backup of CStor volume through Velero, configure `VolumeSnapshotLocation` with provider `openebs.io/cstor-blockstore`. Sample YAML file for volumesnapshotlocation can be found at `example/06-volumesnapshotlocation.yaml`.
```

Comment:
To take remote backup of cStor volume snapshot to cloud or S3 compatible storage, configure `VolumeSnapshotLocation` with provider `openebs.io/cstor-blockstore`. 

Can you provide samples for popular use cases like: 
- Minio
- AWS S3
- GCP buckets. 

They could be different commented out YAMLs in the same snapshotlocation example yaml.

Reply:
Throughout this document - mention remote/cloud backup and local backup.

Reply:
ok